core-bundle/src/Resources/contao/modules/Module.php

```diff
@@ -257,7 +257,7 @@ abstract protected function compile();
 	 */
 	protected function getResponseCacheTags(): array
 	{
-		return array(System::getContainer()->get(EntityCacheTags::class)->getTagForModelClass($this->objModel));
+		return array(System::getContainer()->get(EntityCacheTags::class)->getTagForModelInstance($this->objModel));
```

Comment:
Should we also check that `$this->objModel` is not null?

Reply:
Changed in b7ca2d1d0bb00eb434a5321153cdec05eec9f120.

Reply:
How can this be null? The property is annotated with `@var ModuleModel` as well. On the other hand: the constructor type hints are wrong as well.